bugfix: [Scala 2] Automatically import types in string interpolations

tgodzik opened the pull request:
Previously, when trying to provide inside a string we would not get completions from the workspace. Now, we properly add them.

Related to https://github.com/scalameta/metals/issues/2105 but it still needs a fix for Scala 3

tanishiking reviewed (approved):
LGTM!

Just a few questions, but they won't be blockers for merging this PR :)

- Is there an option or something to invoke interpolation completion without triggering completions manually?
  - I trigger interpolation completion by hand using "^+Space" (for me), but it would be useful if interpolation completion will be triggered automatically when we write `$xxx` inside of `""`.

![interpolator](https://user-images.githubusercontent.com/9353584/178922263-50dec91d-bc8d-4124-9c8b-bb425dbf4394.gif)

---

Also, is it possible to extend this fix to normal completion in`s""`?

![interpolator-s](https://user-images.githubusercontent.com/9353584/178923207-771191e9-e683-4b81-87f9-fcb4fe3f8f76.gif)

I usually write `s""` and start writing string interpolation, so it would be great if this fix is available in `s""` :)

tgodzik commented:
> LGTM!
> 
> Just a few questions, but they won't be blockers for merging this PR :)
> 
> * Is there an option or something to invoke interpolation completion without triggering completions manually?
>   
>   * I trigger interpolation completion by hand using "^+Space" (for me), but it would be useful if interpolation completion will be triggered automatically when we write `$xxx` inside of `""`.

That's controlled by `editor.quickSuggestions`  in VS Code, you can change it for the comments

> Also, is it possible to extend this fix to normal completion in`s""`?
> I usually write `s""` and start writing string interpolation, so it would be great if this fix is available in `s""` :)

Fixed!

I also changed the label to include the signature of the method for InterpolationMemberCompletions, which makes it look like all the other methods. No need to make it special.

See the two added commits for detals.

dos65 reviewed (approved):
:+1:
